src/controllers/home.ts

```diff
@@ -88,7 +93,8 @@ export const startCharging = async (req: Request, res: Response) => {
     folioId: folio.id,
     amount: demoTariff.reservationAmount,
     currency: demoTariff.currency,
-    subject: `Wirelane Charging Session ${chargingSession.id} at ${evseId} on 2021-03-25 at 2.37 pm - Tariff: €${demoTariff.pricePerKwh}/kWh`,
+    //subject: `Wirelane Charging Session ${chargingSession.id} at ${evseId} on 2021-03-25 at 2.37 pm - Tariff: €${demoTariff.pricePerKwh}/kWh`,
+    subject: `Wirelane Charging Session: 33,4 kWh * €${demoTariff.pricePerKwh} – wrln.de/maseven/123456”`
```

Comment:
The kWh from the charging session should be used, instead of hardcoding `33,4` here. Also the wrln.de URL is incorrect, we don't have a hotel path in there.
```suggestion
    subject: `Wirelane Charging Session: ${chargingSession.kWh} kWh * €${demoTariff.pricePerKwh} – wrln.de/${chargingSession.id.substring(0, 8)}`
```